File: src/main/java/de/huwi/liberatepdf2/restservice/PdfDTO.java
package de.huwi.liberatepdf2.restservice;

import org.springframework.web.multipart.MultipartFile;

import lombok.ToString;

/**
 * A PDF file
 */
@ToString
public class PdfDTO {
	/**
	 * Byte content of the PDF
	 */
	private MultipartFile file;

	/**
	 * File name of the PDF
	 */
	private String password;

	public MultipartFile getFile() {
		return this.file;
	}

	public String getPassword() {
		return this.password;
	}

	public void setFile(final MultipartFile file) {
		this.file = file;
	}

	public void setPassword(final String password) {
		this.password = password;
	}
}

File: LiberatePDF2-Rest-Service/src/main/java/rocks/huwi/liberatepdf2/restservice/PdfDTO.kt
package rocks.huwi.liberatepdf2.restservice

import lombok.ToString
import org.springframework.web.multipart.MultipartFile

/**
 * A PDF file
 */
@ToString
class PdfDTO {
    /**
     * Byte content of the PDF
     */
    var file: MultipartFile? = null

    /**
     * File name of the PDF
     */
    var password: String? = null
}